build/resolver/svn/Dockerfile

```diff
@@ -9,6 +9,13 @@ RUN apt-get update \
     && apt-get install -y subversion \
     && rm -rf /var/lib/apt/lists/*
 
+RUN apt-get clean \
```

Comment:
Why not merge this with the previous `RUN`?

Reply:
Have fixed. PTAL